client/src/pages/Welcome.tsx

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import styled from "styled-components";
+import logo from "../assets/budgety_logo.png";
+import { Link } from "react-router-dom";
+
+const Container = styled.div`
```

Comment:
Same thing here with creating a separate welcome.styles.TS file.

Reply:
Ah I see, so instead of using styled components inside the Welcome page component itself, just create a separate file that holds all the styling and apply as needed in the Welcome page, correct?